src/Microsoft.PowerShell.Commands.Diagnostics/ExportCounterCommand.cs

```diff
@@ -65,12 +65,13 @@ public string Path
                 ValueFromPipelineByPropertyName = false,
                 HelpMessageBaseName = "GetEventResources")]
         [ValidateNotNull]
+        [ValidateSet("blg", "csv", "tsv")]
```

Comment:
The file types are explained in the 'Description' section of the documentation.  Did you want them in the 'Parameter' section as well?